crates/graphql_artifact_generation/src/import_statements.rs

```diff
@@ -51,3 +51,18 @@ pub(crate) fn param_type_imports_to_import_statement(
     }
     output
 }
+
+pub(crate) fn param_type_imports_to_import_param_statement(
```

Comment:
Maybe we should consolidate generated type names and filenames so we can use just one function here and pass `RESOLVER_PARAM_TYPE` for example